apis/apiextensions/v1alpha1/infrastructuredefinition_types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
```

Comment:
I think we mostly import this as `corev1` elsewhere. Can we do that here (and elsewhere in this PR) for consistency?